recipes/dm-sonnet/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "dm-sonnet" %}
+{% set version = "1.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmind/sonnet/archive/v{{ version }}.tar.gz
+  sha256: 917b50e6d535d2eb2fa5c7f211b30042f2d56d1564560bd79685ce98b156cf51
+
+build:
+  number: 0
+  # skipping windows for now
+  skip: true  # [win]  
+  skip: true  # [win and vc<14]
+  script:
+    - protoc %SRC_DIR%\\sonnet\\protos\\module.proto --python_out=%SRC_DIR%\\sonnet\\protos -I=%SRC_DIR%\\sonnet\\protos  # [win]
+    - rmdir build  # [win]
+    - rm -rf build  # [osx]
+    - protoc $SRC_DIR/sonnet/protos/module.proto --python_out=$SRC_DIR/sonnet/protos -I=$SRC_DIR/sonnet/protos  # [unix]
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - protobuf
```

Comment:
Should protobuf be a host requirement?

Reply:
I'm not sure but isn't it basically a "compiler" that only does codegen?

Reply:
Isn't it a message passing interface?

Reply:
the ``protobuf`` is necessary to use the ``protoc``. ``protoc`` is a "compiler" which generates python code based on the ``proto`` files. It is not really a compiler, but I let in the build section because it is basically a tool which I need before to build the project. Should I let it in ``build`` or ``host``?